### PR TITLE
Removed testgrid annotations from non-compatible jobs.

### PR DESCRIPTION
**Pull request by fgimenez**
Also added new periodics that run the main provider lanes 3 times per day.
They will be added to the 'periodics' dashboard, which includes the
conformance results too.

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 1]
```diff
@@ -387,3 +389,102 @@ periodics:
...
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "export TARGET=k8s-1.20 && automation/test.sh"
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:418]: nit: make this a little more agnostic, i.e: 
```suggestion
          - "export TARGET=$(ls cluster-up/cluster/ | grep -E '^k8s-1\.[0-9]+$' | sort --sort=version -r | sed -n '1p') && automation/test.sh"
```

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 2]
```diff
@@ -387,3 +389,102 @@ periodics:
...
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "export TARGET=k8s-1.19 && automation/test.sh"
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:451]:
```suggestion
          - "export TARGET=$(ls cluster-up/cluster/ | grep -E '^k8s-1\.[0-9]+$' | sort --sort=version -r | sed -n '2p') && automation/test.sh"
```

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 3]
```diff
@@ -387,3 +389,102 @@ periodics:
...
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "export TARGET=k8s-1.18 && automation/test.sh"
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:484]:
```suggestion
          - "export TARGET=$(ls cluster-up/cluster/ | grep -E '^k8s-1\.[0-9]+$' | sort --sort=version -r | sed -n '3p') && automation/test.sh"
```

**Review by dhiller — APPROVED** [review]
/lgtm
/approve
/hold

One nit pick I have. See comments.

File: github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml (hunk 1 above)
Comment (fgimenez) [reply github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: > nit: make this a little more agnostic, i.e:

awesome, much better! :heart_eyes:

**Comment by fgimenez** [comment]
Proposed changes to testgrid here https://github.com/kubernetes/test-infra/pull/20469

**Comment by rmohr** [comment]
/lgtm

executing the jobs once per day sounds good. Nice work.

**Pull request merged**